README.md

```diff
@@ -89,12 +89,12 @@ It's great for prototyping.
 The [Wiki](https://github.com/oilshell/oil/wiki) has many developer docs.  Feel
 free to edit them.  If you make a major change, let us know on Zulip!
 
-There are also READMEs in some subdirs, like `opy/` and `mycpp/`.
+There are also READMEs in some subdirectories, like `opy/` and `mycpp/`.
 
 If you're confused, the best thing to do is to ask on Zulip and someone should
 produce a pointer and/or improve the docs.
 
-Docs for **end users** are linked from each [release
+Docs for **end-users** are linked from each [release
```

Comment:
Hm I don't think this should be hyphenated

I think when it's a noun, there should be no hyphen.  Arguably when it's used as an adjective it can have a hyphen.

https://ell.stackexchange.com/questions/116816/end-user-vs-end-user-correct-usage-of-hyphen